src/universe/UniverseViewpointAdapter.java

```diff
@@ -0,0 +1,59 @@
+package universe;
```

Comment:
It would be nice if the `src/universe/UniverseViewpointAdaptor.java` to `src/universe/UniverseViewpointAdapter.java` renaming (notice o vs e in Adapter), would be a move in the git history, to show what changed in the implementations.